omnigibson/maps/map_base.py

```diff
@@ -55,4 +55,5 @@ def world_to_map(self, xy):
             xy: 2D location in world reference frame (metric)
         :return: 2D location in map reference frame (image)
         """
-        return np.flip((np.array(xy) / self.map_resolution + self.map_size / 2.0)).astype(int)
+        point_wrt_map = th.tensor(xy) / self.map_resolution + self.map_size / 2.0
+        return th.flip(point_wrt_map, dims=tuple(range(point_wrt_map.dim()))).int()
```

Comment:
All these `np.flip` -> `th.flip` conversion makes me very nervous, can you make sure to test these carefully? I think the original code is pretty badly written also :(